source/guide_kirby.rst

```diff
@@ -0,0 +1,161 @@
+
+Kirby was released for the first time in 2009 and is actively developed by the Bastian Allgeier GmbH. Although the code is open source, a license is required for use in production. 
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of 
+
+  * PHP_
+  * domains_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1: 
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+The installation of Kirby is quite easy. Just clone the Kirby Starterkit in your `document root`_. 
+
+.. code-block:: console
+ :emphasize-lines: 2
+
+ [isabell@stardust ~]$ cd ~/html
+ [isabell@stardust html]$ git clone --recursive https://github.com/getkirby/starterkit.git ~/html
+  Cloning into 'html'...
+  remote: Counting objects: 3749, done.
+  remote: Compressing objects: 100% (69/69), done.
+  remote: Total 3749 (delta 25), reused 92 (delta 25), pack-reused 3649
+  Receiving objects: 100% (3749/3749), 7.74 MiB | 9.97 MiB/s, done.
+  Resolving deltas: 100% (1035/1035), done.
+ [isabell@stardust html]$
+
+That's all the magic. If you visit your previously set up domain you should see a working website with demo contents from the Starterkit. 
+
+.. note:: While we are cloning Kirby from Github you can alternatively just download_ the zip and upload the contents manually to your `document root`_.
+
+To simplify future updates we are replacing the ``kirby`` and ``panel`` folders and add them again as Git submodules. Using this workflow, updates will need only two commands on the command line. 
+
+First, we remove the existing folders from the filesystem. Then we stage the changes in Git to remove the folders from the repository index. 
+
+.. code-block:: console
+ :emphasize-lines: 2,3
+
+ [isabell@stardust ~]$ cd ~/html
+ [isabell@stardust html]$ rm -R panel kirby
+ [isabell@stardust html]$ git add .
+ [isabell@stardust html]$
+
+Finally we are adding the components as Git submodules and try to update them just to be safe.
+
+.. code-block:: console
+ :emphasize-lines: 1,5,9
+
+ [isabell@stardust html]$ git submodule add https://github.com/getkirby/kirby.git
+  Cloning into '/var/www/virtual/isabell/html/kirby'...
+  remote: Counting objects: 4344, done.
+  [...]
+ [isabell@stardust html]$ git submodule add https://github.com/getkirby/panel.git
+  Cloning into '/var/www/virtual/isabell/html/panel'...
+  remote: Counting objects: 11083, done.
+  [...]
+ [isabell@stardust html]$ git submodule update --init --recursive
+ [isabell@stardust html]$
+
+Configuration
+============
+
+When you're ready to launch your website with Kirby you need to purchase a license_. To use your license go to ``/site/config/config.php`` and fill in your licence key.
```

Comment:
This sounds like it can be done in the browser, but the code-block below suggests the file needs to be edited. If the latter is the case, please change the wording to 'edit /var/www/virtual/$USER/html/site/config/config.php'.